CONTRIBUTING.md

```diff
@@ -340,6 +340,15 @@ labels.
   have a new feature you'd like to add, you can start a conversation [in our
   development community](https://zulip.com/development-community/#where-do-i-send-my-message)
   explaining the feature idea and the problem that you're hoping to solve.
+- **I'm waiting for the next round of review on my PR. Can I pick up another
+  issue in the meantime?** Someone's first Zulip PR often requires quite a bit
+  of iteration, so please go through at least one round of feedback before
+  picking up a second issue. After that, sure! If
+  [Zulipbot](https://github.com/zulip/zulipbot) does not allow you to claim an
+  issue, you can post a comment describing the status of your other work on the
+  issue you're interested in, and asking for the issue to be assigned to you.
+  Note that addressing feedback on in-progress PRs should always take priority
+  over starting a new PR.
```

Comment:
The only potential addition I can think of would be an example comment to post, and I'm not sure how much of a value-add it would actually be, but in case anyone finds it worthwhile:

```suggestion
  over starting a new PR.
  
  For example, while awaiting a decision on PR #1234, to adopt issue #5678, leave a comment on #5678 such as:
  > Hi, while awaiting another round of feedback on #1234, I'd like to work on this issue that I feel confident in solving. Could someone in this thread assign this issue to me since Zulipbot will not?
```

Reply:
Thanks! I'm inclined not to add such an example, in part because I don't want to see it copy-pasted, rather than the contributor providing individualized details. ;)

Reply:
Yeah, I agree.